### PR TITLE
 [bitnami/kubeapps] Fix permissions for apprepositories issue 5323 tanzu RBAC

### PR DESCRIPTION
**Pull request by cristi-vlad**
### Description of the change

 [bitnami/kubeapps] RBAC
Fix RBAC  permissions for apprepositories issue 5323 tanzu 

### Benefits

Fix error:
```
Missing permissions internal: Unable to fetch app repo "datahub-repo" from namespace "kubeapps": unable to read secret "apprepo-datahub-repo-secrets": secrets "apprepo-datahub-repo-secrets" is forbidden: User "xxxxxxx" cannot get resource "secrets" in API group "" in the namespace "kubeapps".
```
### Possible drawbacks

N/A

### Applicable issues

- Fixes https://github.com/bitnami/charts/issues/26478
- Related to https://github.com/vmware-tanzu/kubeapps/issues/5323

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by dgomezleon — APPROVED** [review]
LGTM. Thanks for contributing

**Comment by dgomezleon** [comment]
Hi @cristi-vlad,

Could you please rebase from `main` to fix an issue with the tests?

**Review by dgomezleon — DISMISSED** [review]

File: bitnami/kubeapps/templates/apprepository/rbac.yaml [hunk 1]
```diff
@@ -212,7 +219,7 @@ rules:
     verbs:
       - '*'
   - apiGroups:
-      - ""
+      - "kubeapps.com"
```
Comment (dgomezleon) [inline bitnami/kubeapps/templates/apprepository/rbac.yaml:222]:
```suggestion
      - kubeapps.com
```
I have just checked that this change breaks our cypress test (placed under `.vib/kubeapps/cypress`) with the following issue.

```
Timed out retrying after 30000ms: Expected to find content: 'bitnamicharts' but never did. 
```
Could you please take a look? You can install the chart with the command below:

```
helm install kubeapps . -f ../../.vib/kubeapps/runtime-parameters.yaml
```

Removing this line makes the test work.

**Review by dgomezleon — COMMENTED** [review]
The issue with the test is introduced by the change.

File: bitnami/kubeapps/templates/apprepository/rbac.yaml [hunk 2]
```diff
@@ -212,7 +219,7 @@ rules:
     verbs:
       - '*'
   - apiGroups:
-      - ""
+      - kubeapps.com
```
Comment (dgomezleon) [inline bitnami/kubeapps/templates/apprepository/rbac.yaml:222]: Hi @cristi-vlad ,

Did you check [this comment](https://github.com/bitnami/charts/pull/26488#discussion_r1619105758)?

Note that this line breaks the tests as I mentioned.

I would suggest to keep this line as it was.
```suggestion
      - ""
```

Could you please check if with this the issue you want to fix is solved too?

**Review by dgomezleon — CHANGES_REQUESTED** [review]
Please validate that the chart works with:

```
helm install kubeapps . -f ../../.vib/kubeapps/runtime-parameters.yaml```

File: bitnami/kubeapps/templates/apprepository/rbac.yaml (hunk 2 above)
Comment (cristi-vlad) [reply bitnami/kubeapps/templates/apprepository/rbac.yaml:222]: Hello. Yes, i applied that change but pipeline still fails. Will apply your proposal and see. Thank you !

**Comment by dgomezleon** [comment]
Thanks @cristi-vlad for applying the changes. A new error appeared with the test, but this is unrelated to your changes. We are working on a fix and I will let you know as soon as possible.

**Comment by dgomezleon** [comment]
The fix was applied in #26605. Could you please rebase `main` to get those changes? Hopefully, it should work afterward.

**Comment by cristi-vlad** [comment]
Too many fixes after rebase. Will open a new one

**Comment by cristi-vlad** [comment]
Created new PR: #26814

**Pull request closed without merging**